cumulusci.yml

```diff
@@ -106,6 +106,22 @@ tasks:
                 - record_type: Opportunity.NPSP_Default
                   default: true
 
+    test_data_bdi:
+        description: 'Generate and load test data for BDI benchmarks into an org'
+        class_path: tasks.generate_bdi_data.GenerateBDIData
+        options:
+            mapping_yaml: 'testdata/bdi_benchmark/mapping.yml'
+            num_records: 2500
```

Comment:
Does this remain for manual testing, or should we remove it now that the robot suite setup does the same thing with a direct reference to the task class?

Reply:
Yeah I left it for manual (including UI) testing.